Readme.md

```diff
@@ -44,27 +80,56 @@ import (
 )
 
 func main() {
-    // Instantiates a client to use send messages to the Rudder API.
-    // User your WRITE KEY in below placeholder "RUDDER WRITE KEY"
-    client, _ := analytics.NewWithConfig(<WRITE_KEY>, <DATA_PLANE_URL>,
+    // Instantiates a client to use send messages to the RudderStack API.
+    
+    // User your write key in the below placeholder:
+    
+    client, _ := analytics.NewWithConfig(WRITE_KEY,
 		analytics.Config{
-			Interval:  30 * time.Second,
-			BatchSize: 100,
-			Verbose:   true,
+			DataPlaneUrl: DATA_PLANE_URL,
+			Interval:     30 * time.Second,
+			BatchSize:    100,
+			Verbose:      true,
+			Gzip:         1,
 		})
 
     // Enqueues a track event that will be sent asynchronously.
+    
     client.Enqueue(analytics.Track{
         UserId: "test-user",
         Event:  "test-snippet",
     })
 
     // Flushes any queued messages and closes the client.
+    
     client.Close()
 }
 ```
 
+## Gzip support
+
+The Go SDK supports Gzip compression from version 4.0.0 and it is set to true by default. However, you can manually enable or disable this feature by setting the `Gzip` parameter to `1` or `0` while initializing the SDK, as shown:
```

Comment:
In the [spec](https://www.notion.so/rudderstacks/Update-Go-SDK-readme-5269984801a345b7849526bc367c53cc), we say the following:
"Gzip is set to true by default" and in the snippet, we have `Gzip: 1`.

However, we say "set 1 to disable and 0 to enable", which is contradictory. Could you please confirm the values, @pallabmaiti

Reply:
Spec updated.